include/aspect/geometry_model/spherical_shell.h

```diff
@@ -179,6 +181,35 @@ namespace aspect
          * Number of tangential mesh cells in the initial, coarse mesh.
          */
         int n_cells_along_circumference;
+
+        /**
+         * The manifold that describes the geometry.
+         */
+        const SphericalManifold<dim> spherical_manifold;
```

Comment:
But then I also need to provide a separate definition in the .cc file to avoid a missing instantiation, is this worth the hassle?